CheckYourEligibility-Admin/Views/Shared/_Layout.cshtml

```diff
@@ -27,6 +26,27 @@
 </head>
 
 <body class="govuk-template__body  js-enabled" data-clarity="@clarityId">
+    <div id="cookie-banner" class="govuk-cookie-banner" role="region" aria-label="Cookies on Check a family's eligibility" data-nosnippet style="display:none;">
```

Comment:
avoid using inline styles, they won't work well due to our xss blocking. Use the classes we discussed earlier, nojsshow and nojshide I believe they were called.